README.md

```diff
@@ -1 +1,56 @@
-# cr-mobile-menu
\ No newline at end of file
+# Mobile Menu
+
+> This project is for practicing code review.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Browser
+- A version control like git
+- A text editor
+
+### Setup
+
+- First clone or download using the link [https://github.com/binhussen/cr-mobile-menu](https://github.com/binhussen/cr-mobile-menu).
+- Navigate to the folder using cd cr-mobile-menu or GUI.
+- Open index.html in your favorite browser.
+- Use your favorite text editor to make a changes on the page, add other content if necessary.
+
+## Authors
+
+👤 Mohammed Hussen (binhussen)
+
+- GitHub: [@binhussen](https://github.com/binhussen)
+- Twitter: [@binhussens](https://twitter.com/binhussens)
+- LinkedIn: [Linkedin](https://www.linkedin.com/in/binhussen/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse Curriculum
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [**OPTIONAL**] You did a great job in customizing your README file, however **microverse** has provided a new [README](https://github.com/microverseinc/readme-template) file. It could be better if you update your project accordingly